### DISCUSSION TITLE
3.0.0-beta12

### DISCUSSION THREAD

**Discussion by Saranya-jena** [litmuschaos/litmus#4174] [Releases]
## What's Changed
* chore(3.0.0-beta10): Adding the manifests for 3.0.0-beta10 by @ispeakc0de in https://github.com/litmuschaos/litmus/pull/4130
* Update .gitattributes to show golang language  by @imrajdas in https://github.com/litmuschaos/litmus/pull/4139
* feat/teaming: Added revamped UI of teaming for litmus 3.0.0 by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4134
* feat: Probes/Added backend implementation for probes as resource by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4116
* Replace deprecated command with environment file by @jongwooo in https://github.com/litmuschaos/litmus/pull/4115
* chore: [mentorship]: Updated mentorship docs by @amityt in https://github.com/litmuschaos/litmus/pull/4152
* feat/gitops/image-registry: Added revamped UI screens for Gitops and image registry by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4135
* feat: Added YAMLBuilder Component by @hrishavjha in https://github.com/litmuschaos/litmus/pull/4150
* Added UI screens for Resiliency probes by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4154
* [ChaosCenter]: Implement api token CRUD logic & frontend view by @namkyu1999 in https://github.com/litmuschaos/litmus/pull/4138
* ci-cd.md: improve markdown formatting by @vazhnov in https://github.com/litmuschaos/litmus/pull/4156
* feat: Updated SideBar Component by @hrishavjha in https://github.com/litmuschaos/litmus/pull/4155
* fix: [chaos-center]: Fixed subscriber status issue and minor experiment types fix in graphql server by @amityt in https://github.com/litmuschaos/litmus/pull/4158
* Update the code architecture to Interface model for Environment pkg by @Freedisch in https://github.com/litmuschaos/litmus/pull/4159
* Fixed issue with mongo watch events in graphql server by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4160
* Fixed issue with update user state by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4164
* Added util to update page title and added empty data check on environment page by @hrishavjha in https://github.com/litmuschaos/litmus/pull/4162
* Added changes required for dex setup for litmus 3.0.0-beta11 by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4163
* Test/mocks: added mocks for test cases by @SohamRatnaparkhi in https://github.com/litmuschaos/litmus/pull/4133
* Test/chaos_experiment_runs: Handler and service tests for experiment runs by @SohamRatnaparkhi in https://github.com/litmuschaos/litmus/pull/4136
* fix: Updated `Account Settings` and `Infrastructure` pages by @hrishavjha in https://github.com/litmuschaos/litmus/pull/4170
* Added 3.0.0-beta12 installation manifests by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4169
* chore(3.0.0-beta11): Adding the manifests for 3.0.0-beta11 by @ispeakc0de in https://github.com/litmuschaos/litmus/pull/4171
* Fixed issue with updating audit fields in recent experiment runs by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4172
* fix: Fixed UI issues found in teaming page by @hrishavjha in https://github.com/litmuschaos/litmus/pull/4173

## New Contributors
* @jongwooo made their first contribution in https://github.com/litmuschaos/litmus/pull/4115
* @vazhnov made their first contribution in https://github.com/litmuschaos/litmus/pull/4156
* @Freedisch made their first contribution in https://github.com/litmuschaos/litmus/pull/4159

**Full Changelog**: https://github.com/litmuschaos/litmus/compare/3.0.0-beta11...3.0.0-beta12

<hr /><em>This discussion was created from the release <a href='https://github.com/litmuschaos/litmus/releases/tag/3.0.0-beta12'>3.0.0-beta12</a>.</em>

**Comment by amoughit** [comment]
Hi, is it normal that now it's mandatory to set a probe otherwise the experiment cannot be launched ?
